transfer.md

```diff
@@ -0,0 +1,51 @@
+# cri-o Usage Transfer
+
+This document outlines useful information for ops and dev transfer as it relates to infrastructure that utilizes cri-o.
+
+## Operational Transfer
+
+## Abstract
+
+The `crio` daemon is intended to provide the [CRI](https://github.com/kubernetes/community/blob/master/contributors/devel/container-runtime-interface.md) socket needed for Kubernetes to use (See the document for [configuring kubernetes to use cri-o](./kubernetes.md) for more information on that).
+Therefore the `crioctl` command line is a client that interfaces to the same grpc socket as the kubernetes daemon would, for talking to the `crio` daemon.
+In many ways `crioctl` is only as feature rich as the Kubernetes CRI requires.
+For this there are additional tools e.g. `kpod` particularly for human interaction.
+
+
+## System Tools
+
+Many traditional tools will still be useful, such as `pstree`, `nsenter` and `lsns`.
+As well as some systemd helpers like `systemd-cgls` and `systemd-cgtop` are still just as applicable.
+
+## Equivalents
+
+For many troubleshooting and information collection steps, there may be an existing pattern.
+Following provides equivalent with cri-o tools for gathering information or jumping into containers, for operational use.
+
+| Existing Step | CRI-O (and friends) |
+| :---: | :---: |
+| `docker info` | [`kpod info`](./docs/kpod-info.2.md) |
+| `docker inspect` | [`kpod inspect`]() (TBD) |
+| `docker stats` | [`crioctl ctr status`]() (TBD) |
```

Comment:
oh, if there is a kpod subcommand, we can use it. That was not in the works when i started this doc.